src/backend/marsha/core/tests/test_serializers_upload_confirm.py

```diff
@@ -22,8 +22,9 @@ def test_serializers_update_state_valid_data(self):
         valid_keys = (
             "{!s}/video/{!s}/0123456789".format(uuid4(), uuid4()),
             "{!s}/timedtexttrack/{!s}/0123456789".format(uuid4(), uuid4()),
-            "{!s}/timedtexttrack/{!s}/0123456789_fr".format(uuid4(), uuid4()),
             "{!s}/timedtexttrack/{!s}/0123456789_fr_cc".format(uuid4(), uuid4()),
+            "{!s}/timedtexttrack/{!s}/0123456789_fr_st".format(uuid4(), uuid4()),
+            "{!s}/timedtexttrack/{!s}/0123456789_fr_ts".format(uuid4(), uuid4()),
```

Comment:
We can avoid forgetting to update the regex the next time we add or modify an option. Something like:
```
["{!s}/timedtexttrack/{!s}/0123456789_fr_cc".format(uuid4(), uuid4(), mode) for mode in for mode, _ in TimedTextModel.MODE_CHOICES]
```